test/disk_usage_test.rb

```diff
@@ -0,0 +1,33 @@
+require 'test_helper'
+require 'rack/test'
+
+require 'smart_proxy_pulp_plugin/pulp_plugin'
+require 'smart_proxy_pulp_plugin/disk_usage'
+
+class DiskUsageTest < Test::Unit::TestCase
+  def test_has_path_should_be_true
+    disk_test = ::PulpProxy::DiskUsage.new(:path => '/etc')
+    assert_equal('/etc', disk_test.path)
+  end
+
+  def test_invalid_path_raises_error
+    assert_raises Sys::Filesystem::Error do
+      ::PulpProxy::DiskUsage.new(:path => '/e/t/c')
+    end
+  end
+
+  def test_has_stat_should_be_true
+    disk_test = ::PulpProxy::DiskUsage.new(:path => '/etc')
+    assert_not_nil(disk_test.stat)
+    assert(disk_test.stat.is_a?(Sys::Filesystem::Stat))
+    assert_not_nil(disk_test.stat.bytes_free)
+  end
+
+  def test_should_return_valid_json
+    disk_test = ::PulpProxy::DiskUsage.new(:path => '/etc')
```

Comment:
Wouldn't it be more useful to test json representation of an the object in a certain state? Atm the test verifies correctness of to_json method...